python/plan.ps1

```diff
@@ -0,0 +1 @@
+. ../python37/plan.ps1
```

Comment:
This will need a minimal amount of metadata. We don't have a windows ci check for it yet, but you can see the required fields here: https://github.com/habitat-sh/core-plans/blob/master/bin/ci/check-default-variables.sh